databricks/koalas/extensions.py

```diff
@@ -188,7 +188,7 @@ def plot(self):
         >>> type(GeoAccessor) # doctest: +SKIP
             <class 'databricks.koalas.extensions.GeoAccessor'>
         >>> kdf = ks.DataFrame({"longitude": np.linspace(0,10),
-        ...                         "latitude": np.linspace(0, 20)})
+        ...                     "latitude": np.linspace(0, 20)})
```

Comment:
@ueshin This the indentation fix you were after?

Reply:
Yes, looks fine.
Could you also fix the other places similar to this?

Reply:
Yep, got it, thanks!